### ISSUE TITLE
Navigation issue - left arrow

### ISSUE THREAD

**Issue by timeinpixels** [VSCodeVim/Vim#8595]
**Describe the bug**
In recent VSCode update, the behaviour of left arrow in insert mode has changed.
Verified it on two separate installations.

**To Reproduce**

1. Have a line of text
2. Position the cursor in the beginning of the line
3. Go to insert mode ('i')
4. Hit cmd - right arrow to go to the end of line
5. Hit left arrow
6. The cursor goes back to the beginning of the line

**Expected behavior**
The cursor should move 1 character to the left (not to the beginning of the line, or position from where we moved with cmd-right arrow).

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 1.26.0
- VSCode version: 1.82.2
- OS: macOS Ventura

**Comment by jaoreir** [comment]
Probably related my issue #8585 
Have you also been experiencing navigation problems with keys other than left arrow? Sometimes the right arrow also glitches out but I haven't been able to reproduce it.

**Comment by timeinpixels** [comment]
Yeah, same happens when you move with cmd-left arrow and right after that you press right arrow. It will go back to the initial position even if it's the whole line rather than just one character. 
It seems as if the arrow undoes the previous movement in this case. 
Only in insert mode though.

**Comment by vann-at-cpcp-dot-tw** [comment]
Same here

Environment:
Extension (VsCodeVim) version: v1.26.0
VSCode version: 1.82.2
OS: macOS 13.5.2

**Comment by ywj0623** [comment]
Same here

Environment:
Extension (VsCodeVim) version: v1.26.0
VSCode version: 1.78.2
OS: macOS 13.2

**Comment by TryerGit** [comment]
@timeinpixels 

Same issue here:


    Extension (VsCodeVim) version: 1.26.0
    VSCode version: 1.74.3
    OS: Linux Ubuntu 2020 LTS

How did you "roll back" / downgrade to previous version? Right clicking on the extension "Install Another Version..." is greyed out on my machine.

**Comment by Sky-iii** [comment]
> @timeinpixels
> 
> Same issue here:
> 
> ```
> Extension (VsCodeVim) version: 1.26.0
> VSCode version: 1.74.3
> OS: Linux Ubuntu 2020 LTS
> ```
> 
> How did you "roll back" / downgrade to previous version? Right clicking on the extension "Install Another Version..." is greyed out on my machine.

you can download v1.25.2 from the "version history" tab on this page https://marketplace.visualstudio.com/items?itemName=vscodevim.vim, and manually install it on vscode using "install from vsix" option.

**Comment by zeromask1337** [comment]
Thank you! Instantly fixed issue. Guess we have to wait for proper fix for some time, considering 1.5 overall number :)

**Comment by joshk0** [comment]
Still seeing this issue with latest version. Downgrade still works fine.

**Comment by dokim-schonfeld** [comment]
Still seeing this issue with the latest version.

**Comment by TryerGit** [comment]
> > @timeinpixels
> > Same issue here:
> > ```
> > Extension (VsCodeVim) version: 1.26.0
> > VSCode version: 1.74.3
> > OS: Linux Ubuntu 2020 LTS
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > How did you "roll back" / downgrade to previous version? Right clicking on the extension "Install Another Version..." is greyed out on my machine.
> 
> you can download v1.25.2 from the "version history" tab on this page https://marketplace.visualstudio.com/items?itemName=vscodevim.vim, and manually install it on vscode using "install from vsix" option.

Just want to point out that there seems to be a limit on max 5 previous releases. So, for e.g., currently, the oldest visible download is 1.26.2

V1.25.2 is lost in cyberspace somewhere!

**Comment by Protowalker** [comment]
Jumping in to say that as of v1.28.1 this is still an issue, though it's inconsistent.

**Comment by jameskim0987** [comment]
> > > @timeinpixels
> > > Same issue here:
> > > ```
> > > Extension (VsCodeVim) version: 1.26.0
> > > VSCode version: 1.74.3
> > > OS: Linux Ubuntu 2020 LTS
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > How did you "roll back" / downgrade to previous version? Right clicking on the extension "Install Another Version..." is greyed out on my machine.
> > 
> > 
> > you can download v1.25.2 from the "version history" tab on this page https://marketplace.visualstudio.com/items?itemName=vscodevim.vim, and manually install it on vscode using "install from vsix" option.
> 
> Just want to point out that there seems to be a limit on max 5 previous releases. So, for e.g., currently, the oldest visible download is 1.26.2
> 
> V1.25.2 is lost in cyberspace somewhere!

Hi @TryerGit, instead of browsing the versions on https://marketplace.visualstudio.com/items?itemName=vscodevim.vim, you can still `install from vsix` of your desired version by browsing through their releases directly from here: https://github.com/VSCodeVim/Vim/releases

Essentially, 
1. browse to https://github.com/VSCodeVim/Vim/releases/tag/v1.25.2 ( or any other version you desire )
2. download the `.vsix` file
3. on VScode run `install from vsix` and use that `.vsix` file
4. Restart extensions

I hope this helps!

**Comment by TryerGit** [comment]
Still encountering this issue in the latest version of VSCode as well as the latest version of VSCodeVim. I am still working with Version 1.25.2 of VSCodeVim since that seems to be the latest version without the issue.

Any possibility of a fix? @J-Fields

Thank you for your time and effort!

**Comment by dkstrong** [comment]
Also experiencing this issue on the latest version, and downgrading my extension version to 1.25.2 fixes it.

VSCode 1.96.4
Windows 10
